src/app/components/shared-components/store/actions/employee-Profile-Details.actions.ts

```diff
@@ -0,0 +1,17 @@
+import { createAction, props } from '@ngrx/store';
+import { EmployeeProfileDetails } from 'src/app/models/hris/EmployeeProfile/employeeProfileDetails.interface';
```

Comment:
Refactor: is it necessary for the .interface to be in its own directory under models/hris since the others are not? Just let me know on this? It looks like per tab we have dto files?